src/tool_operate.c

```diff
@@ -1209,21 +1209,26 @@ static CURLcode single_transfer(struct GlobalConfig *global,
           CURLU *uh = curl_url();
           if(uh) {
             char *updated;
-            if(curl_url_set(uh, CURLUPART_URL, per->this_url,
-                            CURLU_GUESS_SCHEME)) {
-              result = CURLE_FAILED_INIT;
+            CURLUcode uerr;
```

Comment:
You can move the `updated` declaration down to the `else {` clause now... And maybe consider initing it to NULL just to silence the static code analyzer.

Reply:
Done.